api-gateway/src/main/resources/static/scripts/visits/visits.controller.js

```diff
@@ -682,7 +730,73 @@ angular.module('visits')
             }, function () {
                 console.log("Failed to create corresponding bill!");
             });
-        }
+        }*/
+
+        self.someFunction = function() {
```

Comment:
[nitpick] It's always good to give meaningful names to your functions even if they only execute two small logs.